AvaliadorDisciplinas.FRONTEND/src/services/simularPeriodoService.js

```diff
@@ -0,0 +1,9 @@
+import { api } from "./api";
+
+async function getSimulacao(disciplinasIds) {
+    const ids = disciplinasIds.map(id => `courses_id=${id}`).
+        join('&');
```

Comment:
React tá reclamando pra ser na mesma linha.
```suggestion
    const ids = disciplinasIds.map(id => `courses_id=${id}`).join('&');
```